### ISSUE TITLE
[Bug]: Covers report NaN% opening state

### ISSUE THREAD

**Issue by Ultra-bob** [t0bst4r/matterbridge-home-assistant#235]
### What happened?

When I add the cover domain to my configuration, matterbridge starts successfully, and the devices are registered, but they show up as NaN% cover position, which I assume causes Google Home to reject it, so all of my devices show as unavailable, and the covers don't show up at all.

### Relevant log output

```shell
2024-07-29 15:47:35.772 ERROR  InteractionServer    Subscription 3122509070 for Session 8365: Error while sending initial data reports The number NaN cannot be converted to a BigInt because it is not an integer
    at BigInt (<anonymous>)
    at toBigInt (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/util/Number.js:25:38)
    at DataWriter.writeUInt64 (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/util/DataWriter.js:32:54)
    at TlvCodec.writePrimitive (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvCodec.js:312:27)
    at TlvByteArrayWriter.writePrimitive (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvSchema.js:74:14)
    at file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvAny.js:57:18
    at Array.forEach (<anonymous>)
    at AnySchema.encodeTlvInternal (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvAny.js:37:15)
    at file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvArray.js:22:51
    at Array.forEach (<anonymous>
```

### Version

2.3.1

**Comment by t0bst4r** [comment]
Thanks for reporting. 

I’m not at home for three days. I’ll take care on the weekend

**Comment by t0bst4r** [comment]
i have added another null && NaN check. Should be fixed with `2.3.2`
